### ISSUE TITLE
modeling heat loss with a Newton type term for diffusion terms?

### ISSUE THREAD

**Issue by flothesof** [sfepy/sfepy#900]
Hello, 
related to my previous questions #898, I would like to use a term associated to Newton type heat loss on one of my boundaries (see Wikipedia [here](https://en.wikipedia.org/wiki/Newton's_law_of_cooling)).
This would involve adding a surface integral term of the type $\int_\Gamma h (T - T_\infty) d\Gamma$ to the set of equations.
I have searched the code base and the Github for related things but I couldn’t find any.
My question is: can I implement this with the existing terms? Or do I need to define an additional term for this?
I would be willing to contribute an example based on this if I could get it to work...
Thanks again for your help!
Florian

**Comment by flothesof** [comment]
Sorry for the above post, it seems I was too quick to write... I just noticed that the term is actually listed in the docs https://sfepy.org/doc-devel/src/sfepy/terms/terms_dot.html#sfepy.terms.terms_dot.BCNewtonTerm
I’ll try to get it to work then.

**Comment by rc** [comment]
Does it work then?

**Comment by flothesof** [comment]
Yes, it seems to work as expected. Here’s a screenshot with the heat source on top and the heat loss at the bottom. It is a transient Poisson problem with three materials.
![image](https://user-images.githubusercontent.com/2360669/203940585-bc4e4704-e2be-41e0-9a6e-2b59b31e727b.png)

This could be turned into an example if you want.

**Comment by rc** [comment]
Yes, nice examples are welcome! It could replace some of the more "boring" ones.
